README.md

```diff
@@ -151,6 +155,8 @@ For X11 forwarding to work you first need to enable it in your Display Manager a
 connections. To allow connections you can either add an entry for applications or run `xhost +`
 in the console while you are debugging and turn it off again when you are done using `xhost -`.
 
+SSH algorithms that some old embeded devices use may be outdate, there is a compiable method via using `algorithms`. `kex`, `cipher`,`compress`, `hmac`, `serverHostKey` are known to be supported in `algorithms`, data format of these keys is array. Specific content can be referred to [msc/ssh](https://github.com/mscdex/ssh2) (`Client methods`->`connect`->`algorithms`) .
```

Comment:
I suggest the following text adjustment:

> SSH algorithms used by some old embedded devices may be out of date, there is a compileable method using `algorithms`, `kex`, `cipher`,`compress`, `hmac`, `serverHostKey` are known to be supported in `algorithms`. The data format of these keys is array. Supported values can be found in [mscdex/ssh2](https://github.com/mscdex/ssh2/blob/master/README.md#client-methods)(`Client methods`->`connect`->`algorithms`).
which includes a change to the deeplink https://github.com/mscdex/ssh2/blob/master/README.md#client-methods .

Reply:
done